content/posts/2019/2019-10-01-zaznam-setkani.md

```diff
@@ -6,7 +6,12 @@ category: blog
 slug: zaznam-setkani
 description: "Ve středu 25. 9. se v prostorách STRV sešlo na 120 členů a podporovatelů komunity Česko.Digital. Pokud jste se nestihli zúčastnit osobně ani na live streamu, nabízíme záznam všech vystoupení."
 cover: https://i.imgur.com/ObMcG33.jpg
+lang: cs
+langVersion:
+    en: ""
+    cs: ""
 ---
+[//]: # (Do not remove lang and langVersion attributes! Need to be there for successful build)
```

Comment:
Tohle pak můžem eliminovat, až přidám rozhovor s Olivií, je to tak?

Reply:
Ano, je tomu přesně tak. Snažil jsem se vyhledat nějaké řešení, aby build nepadal, pokud daný atribut není ani u jednoho článku, ale moc jsem nepochodil.

Reply:
Jasně, tuhle situaci už jsme tu měli s tagy, během pár commitů ten hack zmizí a je to OK.